sonarapp/Program.cs

```diff
@@ -24,6 +24,8 @@ static void Main(string[] args)
 
             Boolean whatifsssfffjenkinpr = (new verify()).verifywhat("whatifsssfffjenkinpr");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'whatifsssfffjenkinpr' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)